includes/User.php

```diff
@@ -4837,9 +4837,28 @@ public static function makeGroupLinkWiki( $group, $text = '' ) {
 	 * @param $right String Right to query
 	 * @return String Localized description of the right
 	 */
-	public static function getRightDescription( $right ) {
+    public static function getRightDescription( $right ) {
```

Comment:
Do you have code style configured correctly? Formatting seems to be off. Media Wiki has a script for reformatting code, you can find it in the styleguide repo